perception/traffic_light_classifier/src/nodelet.cpp

```diff
@@ -26,6 +26,8 @@ TrafficLightClassifierNodelet::TrafficLightClassifierNodelet(const rclcpp::NodeO
   using std::placeholders::_1;
   using std::placeholders::_2;
   is_approximate_sync_ = this->declare_parameter("approximate_sync", false);
+  backlight_threshold_ = this->declare_parameter<double>("backlight_threshold", 0.85);
```

Comment:
[MUST]
Please do **NOT** set default value in declare_parameter.
Additionally, could you please update README as well?

Reply:
because this node is used as composable node, I removed the default parameter and updated launcher by https://github.com/autowarefoundation/autoware.universe/pull/5852/commits/76d10abed9e9407fd1492fd723040107559e767d
Also, updated README by https://github.com/autowarefoundation/autoware.universe/pull/5852/commits/76d10abed9e9407fd1492fd723040107559e767d